core/src/main/java/org/openconnectors/util/SourceConnectorContext.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.openconnectors.util;
+
+public interface SourceConnectorContext extends ConnectorContext {
```

Comment:
👍  kafka-connect does not provide exposure to the version of the connect framework or the connectors. We use work-arounds i.e

```CONNECTOR_VERSION=x
CONNECTOR_DOC=xxx```

as configurations of the connectors themselves to manage to expose that info

Reply:
What do you recommend ?  
In my mind that specific concern should be take care on the kafka connect adapter layer,  when we get to it

Reply:
Wanted to put the +1 to the `Versionable` as this lays the grounds for exposing versions on the stack

Reply:
+1 to Versionable